velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -221,6 +222,12 @@ class HiveConnectorSplitBuilder {
   }
 
  private:
+  // If passed path is not prefixed with a schema, assume "file:"...
+  static std::string fixUpPath(const std::string& path) {
+    auto rex = std::regex("^[a-z]+[a-z0-9]*:.*$", std::regex::egrep);
+    return std::regex_match(path, rex) ? path : "file:" + path;
```

Comment:
We need this only for regular local paths.
We can avoid regex by checking as below and appending `file:` if it is. 
```
filePath.find("/") == 0
```

Reply:
Done.